src/layers/microscopy-viewer-layer/data-utils/tiff-utils.js

```diff
@@ -0,0 +1,46 @@
+import {fromUrl, Pool, getDecoder } from 'geotiff/dist/geotiff.bundle.min.js';
+
+async function loadTile({image, channel, x, y, pool}) {
+  var tile = await image.getTileOrStrip(x,y, 0, pool)
+  var dataObj = {}
+  const bits8 = image.fileDirectory.BitsPerSample[0]===8
+  const bits16 = image.fileDirectory.BitsPerSample[0]===16
+  const bits32 = image.fileDirectory.BitsPerSample[0]===32
+  dataObj[channel] = (bits8 && new Uint8Array(tile.data)) ||
+                    (bits16 && new Uint16Array(tile.data)) ||
+                    (bits32 && new Uint32Array(tile.data))
+  return dataObj
```

Comment:
Let's use semicolons... I know they're not *really* necessary in JS, but consistency is good.

Reply:
Will run prettier/eslint later